### DISCUSSION TITLE
How do I set an experiment that will test stress on node

### DISCUSSION THREAD

**Discussion by Dorsosiv** [chaos-mesh/chaos-mesh#3879] [Q&A] [answered]
Hey, I am trying to set an experiment as described. But I do not know what to enter here:  

![2023-01-02_12-24](https://user-images.githubusercontent.com/76942886/210219059-5ee84c0f-3df4-4d9f-8310-2d1b21ee3bdb.png)

**Comment by fewdan** [comment, marked as answer]
@Dorsosiv
You can refer to this [link](https://chaos-mesh.org/docs/simulate-physical-machine-chaos/#configuration-description).

Address: Selects the address of Chaosd service to inject faults, only one of address or selector could be specified.
Example: ["192.168.0.10:31767"]

**Reply by Dorsosiv** [reply]
Ok , in my case I deployed the chaos-mesh using helm on K8s. 
Is the chaos-daemon daemonset the same as the chaosd ? 

How do I use them ? 
I tired using their IP addresses combined with the same port you said and it did not work :  
![Screenshot from 2023-01-03 10-05-31](https://user-images.githubusercontent.com/76942886/210319400-be9f2608-32a9-42f0-905a-5f5ef87cdbe4.png)  

I got the error showed above, then I tried to change the work of the controller and the daemonset pods to not use SECURITY_MODE using helm values.yaml file . And still it does not work for me.

Im using it for development env so I do not care about security .  
I would really like your help here thanks :pray: 

Plus I recommend to update the docs in your website a bit more regarding how to make it work (hosts experiments) on K8s.

**Reply by fewdan** [reply]
Have you installed chaosd on the target node? If it is installed, can you try to log in to the node, use "Command-line mode" to see if it can inject faults normally, or use the command line to see if chaosd can work?

**Reply by fewdan** [reply]
If you need to access chaosd through port 31767, you should use chaosd as a server. 
The general process follows this [url](https://chaos-mesh.org/docs/simulate-physical-machine-chaos/#deploy-chaosd-server).
Note:
After deploying Chaosd, before starting Chaosd server, you need to generate a TLS certificate and create a PhysicalMachine in Kubernetes clusters. For more information on how to generate TLS certificates, refer to [Generate TLS certificates for Chaosd](https://chaos-mesh.org/docs/chaosctl-tool/#generate-tls-certificates-for-chaosd).

**Reply by fewdan** [reply]
In addition, it seems that the website is really not easy to use now, and the docs need to be improved.
We are working hard to update the docs.

**Reply by fewdan** [reply]
Also, chaosd and chaos-daemon daemonset are different. chaosd is a tool that needs to be downloaded and deployed separately to inject faults in the environment of physical machines.
While chaos-daemon is an execution component in chaos-mesh, which mainly injects faults by invading the target pod namespace.

**Reply by Dorsosiv** [reply]
About this: "Have you installed chaosd on the target node?" - I did not know that It should be installed on every "target" host (node in the cluster). - must be updated in the docs. 

About this: "Also, chaosd and chaos-daemon daemonset are different. chaosd is a tool that needs to be downloaded and deployed separately to inject faults in the environment of physical machines.
While chaos-daemon is an execution component in chaos-mesh, which mainly injects faults by invading the target pod namespace." 
- PLEASE write that somewhere in the docs where it will be part of the **helm installation**. Plus make sure to write that in **bold**. 

Now its working for us. Thanks :pray:  

You have a great tool. the only thing you need to do is to update the docs :muscle:

**Reply by fewdan** [reply]
@cwen0 Maybe we can update our docs with what the user suggested?
